### PR TITLE
[TASK] #105384 - Remove deprecated AbstractFile::FILETYPE_* constants

### PR DESCRIPTION
**Pull request by brotkrueml**
Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1099
Releases: main

### CODE REVIEW

File: Documentation/ApiOverview/GlobalValues/Constants/Index.rst [hunk 1]
```diff
@@ -88,12 +88,8 @@ File types
 ..  versionchanged:: 13.0
```
Comment (linawolf) [inline Documentation/ApiOverview/GlobalValues/Constants/Index.rst:88]:
```suggestion
..  versionchanged:: 14.0
```

Comment (linawolf) [reply Documentation/ApiOverview/GlobalValues/Constants/Index.rst:88]: You could write ...has been introduced with TYPO3 v13.0. 

It is a versionchanged directive not a versionadded directive. Also version hints for 14.0 will be kept longer then those for 13.0

Comment (brotkrueml) [reply Documentation/ApiOverview/GlobalValues/Constants/Index.rst:88]: Should be a versionadded in the first place IMHO.

Comment (linawolf) [reply Documentation/ApiOverview/GlobalValues/Constants/Index.rst:88]: In my eyes it should have been a deprecation. But it doesnt matter this much, lets move forward

**Review by linawolf — APPROVED** [review]

**Pull request merged**